task_js/index.js

```diff
@@ -62,3 +62,5 @@ function setTasks(tasks) {
     localStorage.setItem("tasks", JSON.stringify(tasks));
```

Comment:
Hi
the pull request should contain all the project files
so i will be able to comment on all files

couldent do it

Reply:
OK,
I upload the files!
sorry i was late.

Reply:
Still not good!